src/truflation/data/exporter.py

```diff
@@ -88,8 +88,38 @@ def export(self, export_details: ExportDetails, df_local: pandas.DataFrame, dry_
                     df_new_data
                 )
 
+            # Ensure primary key is set
+            self.ensure_primary_key(export_details, df_new_data)
+
         return df_new_data
 
+    @staticmethod
+    def ensure_primary_key(export_details: ExportDetails, df_new_data: pandas.DataFrame) -> None:
+        """
+        Ensures that the primary key is created if they do not exist.
+
+        param:
+          export_details: ExportDetails: database details
+          df_new_data: Pandas.DataFrame: dataframe to export
+        """
+
+        engine = create_engine(export_details.url)
```

Comment:
@joequant, @Jarryd-pretorius Please have a look at the code.

I've also added a test case that ensures the primary key is created.

The only concern is this `engine`, I cannot find a proper way to use the `export_details` connector here.